src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE SCHEMA IF NOT EXISTS `books_storage`;
+
+USE `books_storage`;
+
+CREATE TABLE IF NOT EXISTS books (
+   id BIGINT AUTO_INCREMENT PRIMARY KEY,
+   title VARCHAR(50),
```

Comment:
I do not think max 50 length is enough for every book, maybe `255`? that's usually a default value for `varchar`